File: Java/apl-java-exercises/src/br.com.geekuniversity.section07/S07e04.java
package br.com.geekuniversity.section07;

import java.util.Scanner;

public class S07e04 {
    
    public static void main(String[] args) {
        int numberInput = -1, biggestNumber = 0, lowestNumber = -1, averageNumber = 0;

        Scanner userInput = new Scanner(System.in);

        for (int counter = 0; counter < 10; counter++) {
            
            while (numberInput < 0) {
            System.out.printf("Please enter an integer and positive number for index %d: ", counter);
            numberInput = userInput.nextInt();
            }
            
            if (numberInput > biggestNumber) {
                biggestNumber = numberInput;
            }

            if (numberInput < lowestNumber || lowestNumber == -1) {
                lowestNumber = numberInput;
            }

            averageNumber += numberInput;
            numberInput = -1;
        }

        averageNumber = averageNumber / 10;
        System.out.printf("\nBiggest number: %d\nLowest number: %d\nAverage Number: %d", biggestNumber, lowestNumber, averageNumber);

        userInput.close();
    }
}

File: Python/section07/S07e04.py
number = 0
biggestNumber = 0
lowestNumber = 0
averageNumber = 0

number = float(input("Please enter a number: "))
lowestNumber = number
averageNumber += number

for number in range(1, 10):
    number = float(input("Please enter a number: "))
    if range == 0:
        lowestNumber = number
    if number > biggestNumber:
        biggestNumber = number
    elif number < lowestNumber:
        lowestNumber = number
    averageNumber += number

averageNumber = averageNumber / 10

print("\nBiggest number: {0:.2f}\nLowest number: {1:.2f}\nAverage number: {2:.2f}".format(biggestNumber,lowestNumber,averageNumber))